environment.go

```diff
@@ -698,6 +702,10 @@ func (e *envState) initClientConfig() {
 	}
 }
 
+// initSenders initializes the senders that are used to send payloads to
+// external services such as sending GitHub statuses and Jira messages. These
+// are meant to enable specific Evergreen behavior and are distinct from the
+// global application-wide logging system (see (*Settings).GetSender).
 func (e *envState) initSenders(ctx context.Context) error {
```

Comment:
Maybe unnecessary because of the nice comments you put in but should we also change the name of the function to like `initThirdpartySenders` or something to make it clearer?

Reply:
Good idea, renamed it.